File: Blog.Api/Blog.Front/src/models/post.ts
export interface PostResume {
  id: number;
  title: string;
  description: string;
  url: string;
  creationDate: Date;
  publicationDate: Date;
}
export interface UpsertPostData extends PostData {
  markdown: string;
}

export interface PostData {
  title: string;
  description: string;
  url: string;
  html: string;
  publicationDate: string;
}

File: Blog.Context/Entities/Post.cs
using System;

namespace Blog.Context.Entities
{
    public class Post
    {
        public int Id { get; set; }
        public string Title { get; set; }
        public string Description { get; set; }
        public string Url { get; set; }

        public string MarkdownContent { get; set; }
        public string HtmlContent { get; set; }

        public DateTime CreationDate { get; set; }
        public DateTime PublicationDate { get; set; }
    }
}
